crates/dbs-arch/src/lib.rs

```diff
@@ -22,3 +22,24 @@ pub use x86_64::*;
 mod aarch64;
 #[cfg(target_arch = "aarch64")]
 pub use aarch64::*;
+
+use std::fmt;
+
+/// Types of devices that can get attached to this platform.
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Copy)]
+pub enum DeviceType {
+    /// Device Type: Virtio.
```

Comment:
Any consideration about `PCI`?
PCI is actually supported, and will this structure be used for x86?

Reply:
The specific information of the pci device will not be passed to the guest kernel through fdt. We pass the pci root bus information to fdt, and then the guest kernel confirms the specific pci device information through the probe pci root bus, so there is no pci device in DeviceType.
Re-confirmed the code. Currently, only DeviceType is used in aarch64. Therefore, this structure is placed in the aarch64 directory, and the submission has been re-modified.